### DISCUSSION TITLE
Problem transferring repository

### DISCUSSION THREAD

**Discussion by KrisBabb** [community/community#68177] [Enterprise]
### Select Topic Area

Question

### Body

We tried a code transfer to another owner but we got the error below, was within 24 hour timeframe (we did at 1.58pm and the requests were sent at 4.44 the day before so well within the 24 hours), message was as follows:
 
Oops! That repository transfer request is invalid. Did you copy the URL correctly?
 
URLs are exactly the ones in the email so copying not the problem (I tried both clicking on the link and copying and pasting). Also checked the following as per GitHub guide:
 
1) Permissions in the new repository allow us to create new repositories
2) The target repository doesn’t have any repositories or forks of the same name (only the original that was shared with us as collaborators - could that be the rpoblem?)
 
Any ideas why we might be getting this error, could it be the links have expired (but was well within the 1 day time frame).

Any help gratefully received

Mark

**Comment by Durgesh4993** [comment]
https://docs.github.com/en/enterprise-server@3.7/repositories/creating-and-managing-repositories/transferring-a-repository

Please check out this article
